### ISSUE TITLE
Php support?

### ISSUE THREAD

**Issue by alamboley** [mikestead/hx-yaml#8]
Hey, nice project!

Is there any reason why php isn't supported?

**Comment by mikestead** [comment]
@alamboley Only because I've not really used haxe for php. Have you given it a go and it didn't work?

**Comment by alamboley** [comment]
Indeed, I'd some issues. The first one was because of the lack of the php flag, it can be added when cpp and neko are turned on. But then I'd some errors.
